superbench/benchmarks/model_benchmarks/pytorch_gpt2.py

```diff
@@ -53,7 +58,13 @@ def __init__(self, name, parameters=''):
         """
         super().__init__(name, parameters)
         self._config = None
-        self._supported_precision = [Precision.FLOAT32, Precision.FLOAT16]
+        self._fp8_recipe = None
+        self._supported_precision = [
+            Precision.FLOAT32,
+            Precision.FLOAT16,
+            Precision.FP8_HYBRID,
+            Precision.FP8_E4M3,
```

Comment:
Do we add E5M2 as well?

Reply:
can only run with e4m3 (fp/bp f4m3) and hybrid (fp e4m3, bp e5m2), pure e5m2 (fp/bp both e5m2) is not supported in te

Reply:
Got it